spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/ResNet50Perf.scala

```diff
@@ -49,15 +49,21 @@ object ResNet50Perf {
   }
 
   def main(argv: Array[String]): Unit = {
+    System.setProperty("bigdl.disable.mklBlockTime", "true");
     System.setProperty("bigdl.mkldnn.fusion.convbn", "true")
```

Comment:
you should add the new configurations to our document. There's one page for Java configuration